packages/common/src/tools/timer/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface Timer {
+  now: () => number
+  setTimeout: (callback: () => void, timeMs: number) => number | NodeJS.Timeout
+  clearTimeout: (timeout: number) => void
```

Comment:
```suggestion
  setTimeout: (callback: () => void, timeMs: number) => ReturnType<typeof setTimeout>
  clearTimeout: (timeout: ReturnType<typeof setTimeout>) => void
```

Reply:
Won't this require us to return NodeJS.Timeout from InMemoryTimer?